frontend/leaderboard/src/Upload.re

```diff
@@ -155,61 +155,52 @@ let updatePoints = (metricsMap, pkUsernameMap, sheetsData) => {
   };
 };
 
-let uploadPoints = (fileCredentials, metricsMap) => {
-  let {
-    Types.FileCredentials.clientId,
-    clientSecret,
-    redirectURI,
-    spreadsheetId,
-  } = fileCredentials;
+let uploadPoints = metricsMap => {
+  let auth = {
+    Bindings.GoogleSheets.scopes: [|
+      "https://www.googleapis.com/auth/spreadsheets",
+    |],
+  };
+  let client = Bindings.GoogleSheets.googleAuth(auth);
+  let spreadsheetId = "";
```

Comment:
I'd imagine we'll probably want to read these sheets ids in through env vars as well